### PR TITLE
Github CI tests run on multiple Python versions

### LINKED ISSUE #112: Add GitLab CI to run tests on supported Python versions
Tests are run currently only on Python3.7, it would be great to have workflows for each supported Python version.

For now, it would mean: 3.7, 3.8, 3.9 and 3.10.

Coverage could be set for specific Python version, for example, 3.9.

### PR DESCRIPTION
**Pull request by markheik**
Resolves: #112

### CODE REVIEW

**Comment by markheik** [comment]
Codecov decreased duo changing coverage from Python3.7 to Python3,9

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,20 +9,28 @@ on:
...
       matrix:
-        python-version: ["3.7"]
+        include:
+        - name: py37
+          python: 3.7
+        - name: py38
+          python: 3.8
+        - name: py39
+          python: 3.9
+          coverage: "--cov"
```
Comment (tobiasah) [inline .github/workflows/ci.yml:25]: why `--cov` wouldn't `true` be enough and more clear?

Comment (markheik) [reply .github/workflows/ci.yml]: Good point. Changed it to `true`

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**